lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -316,8 +327,10 @@ class CategoryRenderer {
     const manualAudits = auditRefs.filter(audit => audit.result.scoreDisplayMode === 'manual');
     const nonManualAudits = auditRefs.filter(audit => !manualAudits.includes(audit));
 
-    /** @type {Object<string, {passed: Array<LH.ReportResult.AuditRef>, failed: Array<LH.ReportResult.AuditRef>, notApplicable: Array<LH.ReportResult.AuditRef>}>} */
+    /** @type {Object<string, AuditsByState>} */
     const auditsGroupedByGroup = {};
```

Comment:
`ByState` is very nice, let's rename this much less nice variable name I think I'm responsible for to `auditsKeyedByState` too :D